### PR TITLE
Add keys to Overlay

### PR DESCRIPTION
**Pull request by watt**
Adding disambiguation keys to Overlay, like we currently have in other stack-likes.

### CODE REVIEW

File: BlueprintUI/Sources/Layout/Overlay.swift [hunk 1]
```diff
@@ -10,28 +10,39 @@ import UIKit
...
         elements: [Element] = [],
         configure: (inout Overlay) -> Void = { _ in }
     ) {
-        self.elements = elements
+        children = elements.map { Child(element: $0) }
         configure(&self)
     }
 
     /// Adds the provided element to the overlay.
+    @available(*, deprecated, renamed: "add(child:)")
```
Comment (kyleve) [inline BlueprintUI/Sources/Layout/Overlay.swift:25]: IMO keeping this and not deprecating it is probably fine; since not everyone needs/wants keys?

Comment (kyleve) [reply BlueprintUI/Sources/Layout/Overlay.swift:25]: (Eg, so the shorter method is probably fine to avoid a ton of deprecation warnings)

**Review by kyleve — APPROVED** [review]

File: BlueprintUI/Sources/Layout/Overlay.swift (hunk 1 above)
Comment (kylebshr) [reply BlueprintUI/Sources/Layout/Overlay.swift:25]: meh, stacks use `add(child:)` and I kinda like the consistency? no strong opinion though

**Review by kylebshr — APPROVED** [review]

**Review by n8chur — APPROVED** [review]

File: BlueprintUI/Sources/Layout/Overlay.swift (hunk 1 above)
Comment (watt) [reply BlueprintUI/Sources/Layout/Overlay.swift:25]: I thought it was confusing to have both `add(_:)` and `add(child:)`, and the latter is consistent with other stacks.

**Pull request merged**